### ISSUE TITLE
Generalized Lambda Distribution?

### ISSUE THREAD

**Issue by azev77** [gecrooks/fieldguide#5]
Unless I missed it, I don't think your guide has the Generalized Lambda Distribution (GLD).
For examples: https://arxiv.org/pdf/1907.06336.pdf
This four-parameter distribution is also pretty flexible.

**Comment by gecrooks** [comment]
Yup, missed those too. (The fundamental problem is, of course, that there seems to be a semi-infinite number of named probability distributions.)

**Comment by azev77** [comment]
Probably, but I feel like there is very finite number of Flexible "generalized" 5 parameter distributions in the literature. 

Also, you prob seen: https://arxiv.org/abs/1706.06889
